.gitlab-ci.yml

```diff
@@ -75,13 +75,13 @@ variables:
     description: |
       The Git revision of the ORT configuration repository to use.
   ORT_DISABLE_ADVISOR:
-    value: ""
```

Comment:
The ORT_DISABLE_ADVISOR, ORT_DISABLE_EVALUATOR and ORT_DISABLE_SCANNER are intentionally left empty else you can not set them from your CI/CD variables for your jobs. Default values are set in https://github.com/oss-review-toolkit/ort-gitlab-ci/blob/main/scripts/set-defaults.sh#L9-L11.

I disable the Advisor as ourdefault advisory provider is OSS Index (see [.gitlab-ci.yml#L128](https://github.com/oss-review-toolkit/ort-gitlab-ci/blob/c17059115e81e29a17a91c3b79f48b593526f8b7/.gitlab-ci.yml#L128)) which requires the ORT user to accept their terms and conditions. Let me shoot in a PR to change our default to OSV and then we can enable the advisor by default.

ORT_DISABLE_SCANNER was set to true by default as not every ORT user might want to scan the sources and scanning can take a long time which is not good for a first time user  experience.

Reply:
Created the PR to change to use OSV by and enable the advisor by default, see https://github.com/oss-review-toolkit/ort-gitlab-ci/pull/22